template/4-value_proposition.md

```diff
@@ -1,16 +1,24 @@
 # The Value Proposition
 
-*Describe the value proposition and argue that it is:*
+Proxima provides a solution to this problem by uniquely combining content sharing and outdoor experiences in an intuitive and easy-to-use mobile application. The concept is simple: to share or consume content, users must physically go to the specified location to access it. This encourages real-world exploration and interaction, promoting a healthier balance between virtual engagement and physical activity.
 
-*1. Easy to communicate*
+On the user side, the incentive for using the application emerges through two main components:
 
-- Simple concept: find virtual posts in the real world
+- Sharing content based on a common context
+- Discovering content in a treasure-hunter fashion
 
-*2. Defensible*
+In Proxima, all content—posts, comments, and interactions—is intrinsically linked to the location where it was created. This establishes a shared context among users, which is unique to Proxima. Unlike other social networks, where context is often artificially created through threads or hashtags, Proxima ensures that users sharing content have a common "background" based on their physical presence at the same location. This commonality fosters a sense of connection between users, enabling them to interact based on shared experiences.
```

Comment:
`, enabling them to interact based on shared experiences.`
I think that we don't have the same experiences as we can be at different time during the year. So I would maybe end the sentence at `between users` .

Tell me what you think

Reply:
That is a good point indeed !
I was thinking of maybe add something like `based on shared location` but I realised that we already insist a lot on the location. Thus I believe that stopping the sentence at `between users` is a good choice.

Done in 73a17280c3d6f41e891b72ef747c8356d4e37b67.

Thank you !